### PR TITLE
feat: add emergency warnings to the FAQ root

### LINKED ISSUE #557: Add possibility to show emergency warnings
screen shot 2018-07-19 at 12 29 58

Acceptance criteria
- Emergency warning will be special article in knowledgebase api
- if the article is present, it means there is some emergency to be displayed to customer
- if the article is not present, it means there is no warning to be displayed

Note GraphQL will need to be updated too.

Zeplin: https://zpl.io/V1qE7ZJ

### PR DESCRIPTION
**Pull request by jaroslav-kubicek**
closes #557<br/><br/><br/><url>LiveURL: https://smartfaq-add-emergencies.surge.sh</url>

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -110,7 +112,7 @@ class App extends React.PureComponent<Props, State> {
   };
 
   renderApp() {
-    const route = this.props.route;
+    const { route, emergencies, language, onClose } = this.props;
```
Comment (jalbertsr) [inline src/App.js]: In the [MR in gitlab](https://gitlab.skypicker.com/frontend/frontend/merge_requests/5695/diffs#9df85bf500d7d18a4217b99ce1d61e635fbd2479_63_75) you're calling this prop `banners` and here `emeregencies`

File: src/index.js [hunk 2]
```diff
@@ -28,6 +29,11 @@ const user = {
   lastname: 'Doe',
 };
 
+const emergencies = [
+  'Because of the middle age ways how to get to the Prague Airport and all donkeys are at strike, we are receiving far more contacts than usual. Please solve your request in our shiny mobile app.',
```
Comment (vborszeki) [inline src/index.js]: 😄

File: src/App.js (hunk 1 above)
Comment (jaroslav-kubicek) [reply src/App.js]: thanks, last minute updates 🤦‍♂️   fixed on FE

**Comment by jalbertsr** [comment]
Adjust margins for titles in mobile design
screenshot

**Comment by jaroslav-kubicek** [comment]
thx for noticing, should be fixed now @jalbertsr

**Pull request merged**